style/ruby/README.md

```diff
@@ -3,47 +3,51 @@ Ruby
 
 [Sample](sample.rb)
 
-* Avoid conditional modifiers (lines that end with conditionals).
-* Avoid multiple assignments per line (`one, two = 1, 2`).
-* Avoid organizational comments (`# Validations`).
+* Avoid conditional modifiers (lines that end with conditionals). 36491dbb9
```

Comment:
Yeah, it seems like they autolink in comments, pull request messages, etc, but maybe not in Markdown documents?

Links would be nice, but I'm not sure if it's worth the readability tradeoff in the source document. Thoughts?